src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,232 @@
+} from 'src/components/job-common'
+//  Q4-2022 Disable log-viewing
+//import UriViewer from 'src/components/UriViewer'
+import WDLViewer from 'src/components/WDLViewer'
+import { Ajax } from 'src/libs/ajax'
+import { bucketBrowserUrl } from 'src/libs/auth'
+import { useCancellation, useOnMount } from 'src/libs/react-utils'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import CallTable from 'src/pages/workspaces/workspace/jobHistory/CallTable'
+
+
+const styles = {
+  sectionTableLabel: { fontWeight: 600 }
+}
+
+// Note: this can take a while with large data inputs. Consider memoization if the page ever needs re-rendering.
+const groupCallStatuses = _.flow(
+  _.values,
+  _.flattenDepth(1),
+  _.countBy(a => {
+    const collapsedStatus = collapseCromwellStatus(a.executionStatus, a.backendStatus)
+    return collapsedStatus !== statusType.unknown ? collapsedStatus.id : collapsedStatus.label(a.executionStatus)
+  })
+)
+
+const statusCell = ({ calls }) => {
+  const statusGroups = groupCallStatuses(calls)
+  // Note: these variable names match the id values of statusType (except for unknownStatuses, which will be their labels).
+  const { ...unknownStatuses } = statusGroups
+
+  const makeRow = (count, status, labelOverride) => {
+    const seeMore = !!status.moreInfoLink ? h(Link, { href: status.moreInfoLink, style: { marginLeft: '0.50rem' }, ...Utils.newTabLinkProps },
+      [status.moreInfoLabel, icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })]) : ''
+    return !!count && div({ style: { display: 'flex', alignItems: 'center', marginTop: '0.25rem' } }, [
+      status.icon(),
+      ` ${count} ${!!labelOverride ? labelOverride : status.label()}`,
+      seeMore
+    ])
+  }
+  return h(Fragment, _.concat(
+    ['submitted', 'waitingForQuota', 'running', 'succeeded', 'failed'].filter(
+      s => statusGroups[s]).map(s => makeRow(statusGroups[s], statusType[s])),
+    _.map(([label, count]) => makeRow(count, statusType.unknown, label), _.toPairs(unknownStatuses)))
+  )
+}
+
+const WorkflowDashboard = (({ namespace, name, submissionId, workflowId }, _ref) => {
+  /*
+   * State setup
+   */
+  const [workflow, setWorkflow] = useState()
+  const [fetchTime, setFetchTime] = useState()
+  //Q4-2022 Disable log-viewing
+  //const [showLog, setShowLog] = useState(false)
+
+  const signal = useCancellation()
+  const stateRefreshTimer = useRef()
+
+  /*
+   * Data fetchers
+   */
+
+  useOnMount(() => {
+    const loadWorkflow = async () => {
+      const includeKey = [
+        'end', 'executionStatus', 'failures', 'start', 'status', 'submittedFiles:workflow', 'workflowLog', 'workflowRoot', 'callCaching:result',
+        'callCaching:effectiveCallCachingMode', 'backendStatus'
+      ]
+      const excludeKey = []
+
+      const timeBefore = Date.now()
+      const wf = await Ajax(signal).Cbas.runs(workflowId)
```

Comment:
This is good for this PR, just noting - this is replaced with a call to Cromwell using a new AJAX function in my current ticket.

Reply:
Oh good, I was going to say, I don't think this would give you all the fields you're hoping for...!